src/main/java/spinbox/commands/AddCommand.java

```diff
@@ -21,12 +21,34 @@
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
+import java.util.function.ToDoubleBiFunction;
 
 public class AddCommand extends Command {
     private static final String MODULE_ADDED = "The following module has been added to SpinBox: ";
     private static final String MODULE_NOT_ADDED = "A module with this code already exists in SpinBox.";
     private static final String NON_EXISTENT_MODULE = "This module does not exist.";
     private static final String NOTE_ADDED = "A new note has been successfully added to ";
+    private static final String UNKNOWN_ITEM_TYPE = "Sorry, we do not recognize the type of item you want to add.";
```

Comment:
Something small to take note - use first-person pronoun here so change to "I" instead of "we"